### DISCUSSION TITLE
What info timing.entries section gives in Lighthouse?

### DISCUSSION THREAD

**Discussion by AAMRAPALI** [GoogleChrome/lighthouse#15840] [Q&A] [answered]
For example, if I have to get FCP, which metric should I refer? I have the option to get it from Metrics sections or timing.entries. And why do these values in these two sections differ? I considered that maybe their units are different - seconds/Ms. But then the values are different entirely. Which makes me think that maybe timing.entries conveys some different info like maybe logs of events happening during the page load. But timing.entries also has 'duration' section - What does this duration represent then? 
 1. Metrics
<img width="427" alt="fcp options" src="https://github.com/GoogleChrome/lighthouse/assets/30808650/b58247de-308a-4023-b5aa-acac67b5b804">

 2. Timing.entries 
<img width="355" alt="fcp option2 " src="https://github.com/GoogleChrome/lighthouse/assets/30808650/4b583489-6b3f-4c02-881a-5576ad58b719">

**Comment by adamraine** [comment, marked as answer]
Don't use `Timing.entries`. This is a log of functions run as a part of Lighthouse execution and is not related to the actual metric value.

Using `lhr.audits['metrics']` will yield the actual timing measurments. `firstContentfulPaint` should represent what the Lighthouse report will display and is what you should use most often because it always accounts for throttling.

If you are using [simulated throttling](https://github.com/GoogleChrome/lighthouse/blob/main/docs/throttling.md), `observedFirstContentfulPaint` will represent the *unthrottled* metric value.
